src/test/groovy/io/openliberty/tools/gradle/TestCompileJSP.groovy

```diff
@@ -94,7 +94,7 @@ public class TestCompileJSP extends AbstractIntegrationTest{
         if (nodes.item(0) instanceof Element) {
             Element child = (Element) nodes.item(0)
             String nodeValue = child.getAttribute("javaSourceLevel")
-            Assert.assertTrue("Unexpected javaSourceLevel ==>"+nodeValue, nodeValue.equals("8"))
+            Assert.assertTrue("Unexpected javaSourceLevel ==>"+nodeValue, nodeValue.equals("17"))
```

Comment:
Shouldn't this still be 8? The test project should still use Java 8. We must support our customers who are using older versions of Java. The whole point of this test is to ensure we do the correct thing when the end user is using Java 8 for their app.